source/fundamentals/crud/restful-api-tutorial.txt

```diff
@@ -0,0 +1,384 @@
+.. _csharp-crud-restful-api-tutorial:
+
+=====================================================================
+Tutorial: Create a RESTful API with the {+driver-short+} Driver
+=====================================================================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: crud, code example, tutorial
+   :description: Learn how to use the .NET/C# Driver to create a RESTful API for your application.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this tutorial, you will learn how to create a RESTful API with endpoints that
+perform basic create, read, update, and delete (CRUD) operations across MongoDB
+Atlas.
+
+Prequisites
+-----------
+
+To complete this tutorial, you must have the following:
+
+- An `Atlas account
+  <https://account.mongodb.com/account/register?tck=docs_atlas>`__ with a
+  deployed and configured MongoDB Atlas cluster that is M0 or higher.
+
+- .NET 6.0 or higher `installed <https://dotnet.microsoft.com/en-us/download>`__ on your machine.
+
+To learn how to get started with MongoDB Atlas and how to load sample data, see the
+:atlas:`Get Started with Atlas Guide </getting-started>`. 
+
+This tutorial uses .NET Core 8.0, but you can use any version later than .NET 6.0.
+
+Set Up Your Project
+-------------------
+
+You can create a new .NET Core project using the web application template that
+Microsoft offers. To do this, run the following commands in your terminal:
+
+.. code-block:: bash
+
+   dotnet new webapi -o MongoExample
+   cd MongoExample
+
+To add the {+driver-short+} to your project as a dependency, run the following command:
+
+.. code-block:: bash
+
+   dotnet add package MongoDB.Driver
+
+The preceding commands create a new web application project for .NET core named
+``MongoExample`` and install the latest {+driver-short+}. The template project
+includes some boilerplate files. During this tutorial, you will add to these
+files and remove some of the boilerplate code to create a RESTful API. 
+
+Design a Document Model and Database Service
+--------------------------------------------
+
+In this section, you will create and configure your MongoDB service and define
```

Comment:
S: remove all future tense from this tutorial. It looks like Vale isnt running because you need to merge master into your branch.